app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt

```diff
@@ -1,4 +1,134 @@
 package com.example.theperiodpurse.ui.cycle
 
-class CycleScreen {
+import android.media.Image
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.theperiodpurse.ui.theme.ThePeriodPurseTheme
+
+class CycleScreen : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            ThePeriodPurseTheme {
+                Surface(color = MaterialTheme.colors.background) {
+                    CycleScreenLayout()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun CurrentCycleBox() {
```

Comment:
Something I see the tutorial emphasize on is passing in `modifier: Modifier = modifier` to make the component reusable. Let's follow that convention.